analysis_template/plhld/inference/test.py

```diff
@@ -0,0 +1,111 @@
+# coding: utf-8
+
+"""
+Test inference model.
```

Comment:
We can remove this test inference model since it is already included in columnflow

Reply:
Done 👍